integration/skaffold/helper.go

```diff
@@ -262,6 +269,9 @@ func (b *RunBuilder) cmd(ctx context.Context) *exec.Cmd {
 	if b.repo != "" && command.Flags().Lookup("default-repo") != nil {
 		args = append(args, "--default-repo", b.repo)
 	}
+	if b.profiles != nil && len(b.profiles) > 0 && command.Flags().Lookup("profile") != nil {
```

Comment:
You can simplify the test len(nil) is 0 in Go.
```suggestion
	if len(b.profiles) > 0 && command.Flags().Lookup("profile") != nil {
```